geps/gep-713/index.md

```diff
@@ -1,1157 +1,747 @@
+
+After a few iterations of Gateway API experimenting with this pattern－both, with its own common kinds of meta resources such as the `BackendTLSPolicy`, as well as via multiple implementation-specific kinds of meta resources (see examples of [current use of meta resources](#current-use-of-meta-resources))－ and rounds of discussion such as the one at [kubernetes-sigs/gateway-api/discussions#2927](https://github.com/kubernetes-sigs/gateway-api/discussions/2927), the pattern has been enhanced to its current form.
+
+## Definition
+
+### Meta resources
+
+*Meta resources* are objects that augment the behavior of other objects (*targets*) in a clean and standard way, by declaring additional specification for these targets “from the outside”－i.e., without directly modifying the objects whose behavior the meta resource intends to affect, nor requiring any reference from the objects to the meta resources that affect them.
+
+Meta resources follow a well-known structure that declares clearly a *target* and an *intent*.
+
+#### Meta resources are well-structured CRDs
+
+Meta resources are typically implemented as Custom Resource Definitions (CRDs) that comply with a particular structure. This structure includes fields for specifying references to one or more other objects－called “targets“－whose behavior the instances of the meta resource intend to augment (i.e. declare additional specification), along with resource-specific fields－the “spec proper”－to describe the intended augmented behavior.
+
+While the targets of a meta resource give the meta resource a *context*, the spec proper declares the *intent* within that context.
+
+A typical meta resource looks like the following – example provided based on a hypothetical `ColorPolicy` kind of meta resource:
+
+```yaml
+apiVersion: policies.controller.io/v1
+kind: ColorPolicy
+metadata:
+  name: my-color-policy
+spec:
+  targetRefs: ## target objects whose behaviour to augment
+  - group: gateway.networking.k8s.io/v1
+    kind: Gateway
+    name: my-gateway
+  rules:      ## the "spec proper" describing the intent, i.e. color the traffic blue
+    color: blue
+```
+
+#### Properties of meta resources
+
+##### Targetability (“where”)
+
+Meta resources specify one or more target resources or specific sections of resources, whose behavior the meta resource intends to augment.
+
+##### Semantics (“why”)
+
+Targeting a resource (or section of a resource) must be interpreted within a given semantics that is proper to the meta resource kind.
+
+Two different meta resource kinds that allow targeting resources of a same a given kind X may have very different semantics, not only because the purpose of the two meta resource kinds differ, but because the mechanics of calculating and applying the augment behavior for X can as well be very different between meta resource kinds.
```

Comment:
```suggestion
Two different meta resource kinds that allow targeting resources of a same a given kind X may have very different semantics, not only because the purpose of the two meta resource kinds differ, but because the mechanics of calculating and applying the augmented behavior for X can as well be very different between meta resource kinds.
```